### ISSUE TITLE
Execution Block ID uid://A001/X15a0/Xc8 Sgr_A_st_g_03_TP

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#197]
Sgr_A_st_g_03_TP
[uid://A001/X15a0/Xc8](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/Xc8)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_g_03_TP/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_g_03_TP/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines

  * [ ] SPW16 H13CN
  * [ ] SPW18 H13CO+/SiO
  * [ ] SPW20 HCO+
  * [ ] SPW22 HNCO
  * [ ] SPW24 Cont 1
  * [ ] SPW26 Cont 2

**Comment by eacmills** [comment]
Weblog is not unpacked

**Comment by eacmills** [comment]
SPW 21 has same issue as https://github.com/ACES-CMZ/reduction_ACES/issues/32 

Pre-baseline subtraction:
<img width="716" alt="pre" src="https://user-images.githubusercontent.com/67716646/218168647-2d58e175-8e34-4d0f-8562-4161aaef417b.png">

Post-baseline subtraction:
<img width="699" alt="post" src="https://user-images.githubusercontent.com/67716646/218168667-e2c345ef-33e6-4e6e-b134-f811599a601c.png">

Flagging for re-calibration.

**Comment by eacmills** [comment]
Also question on the 4 different pipeline logs I see-- it seems like only [Extra Weblog Sgr_A_st_g_03_TP_0 -> pipeline-20220810T214436] ran successfully (and a later pipeline run with a different CASA version totally failed). Is this anything to monitor or note?

**Comment by keflavich** [comment]
No, you can ignore those subsequent failures.  I think they're from my auto-pipeline-runner trying to run TP jobs before I figure out how to tell it not to do that (since there's no need to run any pipeline for TP)

**Comment by eacmills** [comment]
Another look at the (very bad) HCO+/SPW 21 baseline fitting:
![spectral_plot_before_subtraction_uid___A002_Xfc8dc8_X8bf4 ms atmcor_Sgr_A_star_ant0_spw21_pol0](https://user-images.githubusercontent.com/67716646/218278884-5b6cbcca-c952-4f1c-87a7-eb7c71f87a15.png)
